### PR TITLE
Clarify release notes wording for Bug 1822513

### PR DESCRIPTION
**Pull request by jottofar**
Current wording has technical specifics of the fix but does not
communicate how the fix helps a user.

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -1585,7 +1585,7 @@ All MongoDB-based samples have been replaced, deprecated, or removed.
 
 * HTTPS signatures retrieved serialized stores, causing potential time outs before the Cluster Version Operator could complete the tasks. Now, external HTTPS signature retrieval is parallel and all stores will be attempted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1840343[*BZ#1840343*])
 
-* The Cluster Version Operator `currentMinor` version was acquired from `cv.Status.History[0].Version`, which contained the version upgraded to and not the current version. Now, the version history is stored in `CompletedUpdate`  and will display the `currentMinor` version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822513[*BZ#1822513*])
+* Z-stream upgrades that use the option `--to-image`, for example `oc adm upgrade --to-image`, are now allowed even when Cluster Version Operator has `upgradeable=false`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822513[*BZ#1822513*])
```
Comment (wking) [inline release_notes/ocp-4-6-release-notes.adoc:1588]: nit: `upgradeable` -> `Upgradeable` ([API docs][1]).

[1]: https://github.com/openshift/api/blob/2ed1db6e15516dd74bcff7050c30ab48c60f03f4/config/v1/types_cluster_operator.go#L171-L175

**Review by wking — APPROVED** [review]
Wording looks good to me, modulo one casing nit.

/assign @codyhoag

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -1585,7 +1585,7 @@ All MongoDB-based samples have been replaced, deprecated, or removed.
 
 * HTTPS signatures retrieved serialized stores, causing potential time outs before the Cluster Version Operator could complete the tasks. Now, external HTTPS signature retrieval is parallel and all stores will be attempted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1840343[*BZ#1840343*])
 
-* The Cluster Version Operator `currentMinor` version was acquired from `cv.Status.History[0].Version`, which contained the version upgraded to and not the current version. Now, the version history is stored in `CompletedUpdate`  and will display the `currentMinor` version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822513[*BZ#1822513*])
+* Z-stream upgrades that use the option `--to-image`, for example `oc adm upgrade --to-image`, are now allowed even when Cluster Version Operator has `Upgradeable=false`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822513[*BZ#1822513*])
```
[the remark this answers is not in the document]
Comment (jottofar) [reply release_notes/ocp-4-6-release-notes.adoc:1588]: I can reword per your direction but I need to fix the last statement which is not correct.

[the remark this answers is not in the document]
Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc:1588]: Sure, please tweak as needed. Thanks!

[the remark this answers is not in the document]
Comment (jottofar) [reply release_notes/ocp-4-6-release-notes.adoc]: Reworded in commit [25068da](https://github.com/openshift/openshift-docs/pull/26837/commits/25068daab8d4b6118109b03c6f1f798e4d087d0e).

**Review by codyhoag — APPROVED** [review]
LGTM

**Pull request merged**